resources/factions/NATO_OIF.json

```diff
@@ -28,14 +28,16 @@
     "SA 342M Gazelle Mistral",
     "Tornado GR4",
     "UH-1H Iroquois",
-	"UH-60A"
+	"UH-60A",
```

Comment:
Spaces, not tabs (and elsewhere).

Reply:
I ran a search replace on all faction yamls and changed tabs to four spaces.